httplib2shim/__init__.py

```diff
@@ -32,8 +32,10 @@
 
 def _default_make_pool(http, proxy_info):
     """Creates a urllib3.PoolManager object that has SSL verification enabled
-    and uses the certifi certificates."""
+    and uses the certifi certificates.
 
+    Args:
+        proxy_info: A function or None."""
```

Comment:
The `_default_make_pool` is a private func called form one place only, it makes no sense to have variable number of args. About the doc see bellow.